chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -210,7 +210,13 @@ export const chatSDKStateCleanUp = (chatSDK: any) => {
 export const endVoiceVideoCallIfOngoing = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAction>) => {
     let callId = "";
     try {
-        const voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
+        let voiceVideoCallingSdk;
+        if(chatSDK.isVoiceVideoCallingEnabled()) {
+            voiceVideoCallingSdk = await chatSDK.getVoiceVideoCalling();
+        }
+        else{
+            return;
+        }
         if (voiceVideoCallingSdk && voiceVideoCallingSdk.isInACall()) {
```

Comment:
Will the widget crash if  `voiceVideoCallingSdk` was `undefined`? Should `voiceVideoCallingSdk` be nullable?

Reply:
Fixed comment marked voiceVideoCallingSdk  as null